src/planwise/component/datasets.clj

```diff
@@ -42,6 +45,12 @@
   (->> (select-datasets-for-user (get-db store) {:user-id user-id})
        (map db->dataset)))
 
+(defn list-datasets-with-import-jobs
+  [store]
+  (->> (select-datasets-with-import-jobs (get-db store))
+       (map db->dataset)
+       (filter (comp some? :import-job))))
```

Comment:
Is this filter really necessary since you're querying for `import_job IS NOT NULL` already?